### PR TITLE
[Feature] Quote Custom Filters

### PR DESCRIPTION
**Pull request by Civolilah**
Here is the screenshot of implemented solution for Quote custom filters:

Screenshot 2022-12-20 at 15 39 42

### CODE REVIEW

File: src/pages/recurring-invoices/common/hooks.tsx [hunk 1]
```diff
@@ -653,19 +653,19 @@ export function useRecurringInvoiceFilters() {
     },
     {
       label: t('active'),
-      value: 'paid',
+      value: 'active',
       color: 'white',
```
Comment (Civolilah) [inline src/pages/recurring-invoices/common/hooks.tsx:657]: I missed this bug on custom filters for recurring invoices so I just changed this. I thought this was a bit much to create a new PR.

**Comment by turbo124** [comment]
@Civolilah 

looks good, just change placeholder to `status`

@beganovich LGTM when translation updated.

**Comment by Civolilah** [comment]
> 

@turbo124 Both things have been updated. Now, we are correctly displaying `expired` status.

File: src/pages/quotes/common/components/QuoteStatus.tsx [hunk 2]
```diff
@@ -39,8 +43,5 @@ export function QuoteStatus(props: Props) {
       return <Badge variant="dark-blue">{t('approved')}</Badge>;
     default:
       return <Badge variant="light-blue">{t('error')}</Badge>;
-      break;
   }
-
-  return <></>;
```
Comment (beganovich) [inline src/pages/quotes/common/components/QuoteStatus.tsx:45]: Same case for the rest, make sure to use enums.

https://github.com/invoiceninja/ui/blob/main/src/common/enums/quote-status.ts#L13

File: src/pages/quotes/common/components/QuoteStatus.tsx [hunk 3]
```diff
@@ -28,9 +28,13 @@ export function QuoteStatus(props: Props) {
   if (props.entity.invoice_id)
     return <Badge variant="green">{t('converted')}</Badge>;
 
+  if (
+    props.entity.status_id === '2' &&
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/pages/quotes/common/components/QuoteStatus.tsx]: Absolutely, great catch. Done!

File: src/pages/quotes/common/components/QuoteStatus.tsx (hunk 2 above)
Comment (Civolilah) [reply src/pages/quotes/common/components/QuoteStatus.tsx:45]: done

**Review by beganovich — APPROVED** [review]
Looks great, thanks 👍

**Pull request merged**